### ISSUE TITLE
bionic arm64 5.4.0-1077-azure cannot generate BTF

### ISSUE THREAD

**Issue by brycekahle** [aquasecurity/btfhub#59]
I'm recording this here for later investigation, but it seems this kernel has invalid DWARF data that `pahole` cannot generate BTF from:
```
[41] FWD (anon) Error emitting BTF type
Encountered error while encoding BTF.
```
Digging through the pahole/libbpf code, it seems a `name` is required for `FWD` types.

If you don't want me to record issues like this here, just let me know.

**Comment by brycekahle** [comment]
Happens for the same kernel in `focal` too

**Comment by rafaeldtinoco** [comment]
> If you don't want me to record issues like this here, just let me know.

No, please, do open issues for broken kernels so others can find the issue if it occurs to them! Thanks!

**Comment by ryuusetou** [comment]
Did you config kernel with CONFIG_DEBUG_INFO_REDUCED?
